scripts/sql_format_check.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+CWD="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+
+cd $CWD/../
+
+find migrations -type f -name "*.sql" -print0 | while read -d $'\0' FILE
```

Comment:
an alternative would be `git ls-files '**/*.sql'` which has the advantage of getting all files that are part of the repo and no others

Reply:
Good call, we might add other SQL files in different folders anyway.

Reply:
Hrm.. for some reason I couldn't get that working with the loop, oh well, not too big a deal.

Reply:
git ls-files outputs \n - separated so you'd have to remove the `-d $'\0'`